embedded-kafka/src/test/java/com/playtika/test/kafka/EmbeddedKafkaTest.java

```diff
@@ -71,6 +75,7 @@ public void shouldSendAndConsumeTransactionalMessage() throws Exception {
                 .isEqualTo(MESSAGE);
     }
 
+    @Disabled("RHEL image doesn't support to simply install tc")
```

Comment:
can you please specify what was changed in kafka image that we can not install tc anymore?

Reply:
Similar to above, the upstream project (Confluent Platform/Kafka) changed the docker images, they're UBI based now, tc ist not available there (in fact there's no package manager at all).